tests/button/buttonview.js

```diff
@@ -53,21 +53,28 @@ describe( 'ButtonView', () => {
 	describe( '<button> bindings', () => {
 		describe( 'class', () => {
 			it( 'is set initially', () => {
-				expect( view.element.classList.contains( 'ck-button' ) ).to.be.true( 'ck-button' );
-				expect( view.element.classList.contains( 'ck-enabled' ) ).to.be.true( 'ck-enabled' );
-				expect( view.element.classList.contains( 'ck-off' ) ).to.be.true( 'ck-off' );
+				expect( view.element.classList.contains( 'ck-button' ) ).to.be.true;
+				expect( view.element.classList.contains( 'ck-enabled' ) ).to.be.true;
+				expect( view.element.classList.contains( 'ck-off' ) ).to.be.true;
+				expect( view.element.classList.contains( 'ck-button-notext' ) ).to.be.false;
 			} );
 
 			it( 'reacts on model.isEnabled', () => {
 				model.isEnabled = false;
 
-				expect( view.element.classList.contains( 'ck-disabled' ) ).to.be.true( 'ck-disabled' );
+				expect( view.element.classList.contains( 'ck-disabled' ) ).to.be.true;
 			} );
 
 			it( 'reacts on model.isOn', () => {
 				model.isOn = true;
 
-				expect( view.element.classList.contains( 'ck-on' ) ).to.be.true( 'ck-on' );
+				expect( view.element.classList.contains( 'ck-on' ) ).to.be.true;
+			} );
+
+			it( 'reacts on model.noText', () => {
+				model.noText = true;
```

Comment:
As previously [mentioned](https://github.com/ckeditor/ckeditor5-ui-default/pull/22/files#r68731450), `model.set( 'noText', true )` is OK here.